package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -26,7 +26,7 @@
 ===============================================================================
 
 :Author: Zhiyi Wu
-:Year: 2017
+:Year: 2017-2018
```

Comment:
The WaterBridgeAnalysis is now reasonably complicated. There should be a section that describes the algorithm for your analysis so that someone can read your high-level algorithmic description and then look at the code and match it to the algorithm. The algorithm can come after the "howto" section and before the classes.

If you have a paper with this analysis, cite it. (And add it via `duecredit`, too.)

Reply:
I have added a short description of how the algorithm is implemented.